<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicTransformWidget</class>
 <widget class="QWidget" name="MusicTransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicTransformWidget</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>475</width>
     <height>359</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>格式转换</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>473</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>363</y>
     <width>455</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputLineEdit">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>72</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>72</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输入路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputLineEdit">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>105</y>
     <width>310</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>105</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输出路径</string>
   </property>
  </widget>
  <widget class="QComboBox" name="formatCombo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="kbpsCombo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>72</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>105</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>转换格式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>比特率(kbps)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>频率(Hz)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="msCombo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="hzCombo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>声道设置(M/S)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transformButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>325</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>立即转换</string>
   </property>
  </widget>
  <widget class="QLabel" name="loadingLabel">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>337</y>
     <width>220</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="folderBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>添加输入目录(用于批量转换)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>236</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>475</width>
     <height>334</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>line_2</zorder>
  <zorder>inputLineEdit</zorder>
  <zorder>label</zorder>
  <zorder>outputLineEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>formatCombo</zorder>
  <zorder>kbpsCombo</zorder>
  <zorder>inputButton</zorder>
  <zorder>outputButton</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>msCombo</zorder>
  <zorder>hzCombo</zorder>
  <zorder>label_6</zorder>
  <zorder>transformButton</zorder>
  <zorder>loadingLabel</zorder>
  <zorder>folderBox</zorder>
  <zorder>listWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
